src/poller.js

```diff
@@ -110,11 +110,15 @@ module.exports = EventEmitter => {
 		}
 
 		getData () {
+			if (this.data) {
+				return this.data;
+			}
+
 			if (this.error) {
```

Comment:
I'm not entirely sure getData should even throw.

Reply:
What would we do instead, log and return the previous data?